content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,143 @@
+    [LightStep Satellite Setup](https://docs.lightstep.com/docs/satellite-setup) for details about setting up satellites.
+
+1.  Ensure sure you have a LightStep access token.
+
+1.  Ensure you can reach the satellite pool at an address in the format `<Host>:<Port>`, for example `lightstep-satellite.lightstep:9292`.
+
+1.  Deploy Istio with the following configuration parameters specified:
+    - `global.proxy.tracer="lightstep"`
+    - `global.tracer.lightstep.address="<satellite-address>"`
+    - `global.tracer.lightstep.accessToken="<access-token>"`
+    - `global.tracer.lightstep.secure=true`
+    - `global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem"`
+
+    If you are installing via `helm template` you can set these parameters using the `--set key=value` syntax
+    when you run the `helm` command. For example:
+
+    {{< text bash >}}
+    $ helm template \
+        --set global.proxy.tracer="lightstep" \
+        --set global.tracer.lightstep.address="<satellite-address>" \
+        --set global.tracer.lightstep.accessToken="<access-token>" \
+        --set global.tracer.lightstep.secure=true \
+        --set global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem" \
+        install/kubernetes/helm/istio \
+        --name istio --namespace istio-system > $HOME/istio.yaml
+    $ kubectl create namespace istio-system
+    $ kubectl apply -f $HOME/istio.yaml
+    {{< /text >}}
+
+1.  Store your satellite pool's certificate authority certificate as a secret in the default namespace.
+    If the Bookinfo application will be deployed in a different namespace, create the secret in that namespace instead.
+
+    {{< text bash >}}
+    $ CACERT=$(cat Cert_Auth.crt | base64) # Cert_Auth.crt contains the necessary CACert
+    $ NAMESPACE=default
+    {{< /text >}}
+
+    ```bash
+    $ cat <<EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: lightstep.cacert
+        namespace: $NAMESPACE
+        labels:
+          app: lightstep
+      type: Opaque
+      data:
+        cacert.pem: $CACERT
+    EOF
+    ```
+
+1.   Deploy the Bookinfo sample application by following [these instructions](/docs/examples/bookinfo/#deploying-the-application).
+
+## Visualize trace data
+
+1.  Create an ingress gateway for the Bookinfo application by following
+    [these instructions](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port).
+
+1.  Verify the previous step's success by confirming that the environment variable `GATEWAY_URL` is
+    populated in your shell.
+
+1.  Send traffic to the sample application.
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Load the LightStep [𝑥]PM web UI.
+
+1.  Navigate to Explorer.
+
+1.  Find the query bar at the top. The query bar allows you to interactively filter results by a Service, Operation, and/or Tag values.
```

Comment:
Do not use and/or, it introduces unnecessary ambiguity, I would also bold the labels for emphasis:
```suggestion
1.  Find the query bar at the top. The query bar allows you to interactively filter results by a **Service**, **Operation**, and **Tag** values.
```